### DISCUSSION TITLE
Documention changes

### DISCUSSION THREAD

**Discussion by RisingSunLight42** [discordjs/discord.js#7937] [Q&A] [answered]
Hello, 

I've just done a new bot with the latest version of Discord.js and discovered that some informations from the docs (like everything related to `.addChoice()` and `.addChoices()`) are outdated.
Is there a way to change it ? I can help if needed.

**Comment by Jiralite** [comment]
Where is that outdated?

**Reply by RisingSunLight42** [reply]
Sorry for the late reply, here's the link :
https://discord.js.org/#/docs/builders/stable/class/SlashCommandStringOption?scrollTo=addChoice

`.addChoice()` doesn't exist anymore and `.addChoices()` doesn't take array but object.

I've seen it's updated in the category main version, but not in stable, and `npm` install a version more recent than stable.

Sorry in advance if my question is useless.

**Reply by Jiralite** [reply, marked as answer]
discord.js uses 0.13.0 of builders. 0.13.0 of builders would currently be main—not stable.

**Comment by sebastianjnuwu** [comment]
Hum!
